Kristina_Beresneva_4_1.py

```diff
@@ -0,0 +1,40 @@
+#Author Kristina Beresneva
+#25.02.2022
+#task: создать калькулятор
+def plus(a, b):                       #определяем функции математических операций
+    print ("Сумма: ", a, "+", b, "=", a + b)
+def minus(a, b):
+    print ("Разность: ", a, "-", b, "=", a - b)
+def umnozh(a, b):
+    print ("Произведение: ", a, "*", b, "=", a * b)
+def div(a, b):
+    print ("Частное: ", a, "/", b, "=", a / b)
+def div1(a, b):
+    print ("Целочисленное деление: ", a, "//", b, "=", a // b)
+def stepen(a, b):
+    print ("Возведение в степень: ", a, "**", b, "=", a ** b)
+def ostat(a, b):
+    print ("Остаток от деления: ", a, "%", b, "=", a % b)
+    
+while True:                  #проводим операции, пока пользователь не введет exit
+    operation = input("Выберите операцию: +, -, *, /, //, **, % ")
+    if operation == "exit":
+        break
+    num1 = int(input("Введите число: "))
+    num2 = int(input("Введите число: "))
+    if operation == "+":
+        plus(num1, num2)
```

Comment:
```suggestion
       print("additional = ", plus(num1, num2))
```
These changes can be applied to all function calls